src/v0/destinations/fb_custom_audience/transform.js

```diff
@@ -105,6 +106,11 @@ const prepareResponse = (
 
   prepareParams.access_token = accessToken;
 
+  if (isDefinedAndNotNullAndNotEmpty(appSecret)) {
+    prepareParams.appsecret_time = Math.floor(Date.now() / 1000); // Get current Unix time in seconds
```

Comment:
get the `Date.now()` and store it in a variable and use same in `appsecret_time` and in the `generateAppSecretProof`.

Current logic might work well with seconds precision but in future if we add milliseconds precision the timestamps might vary

Reply:
addressed the comment